test/measurement_calibration/generate_data.py

```diff
@@ -0,0 +1,226 @@
+    ghz_circ.cx(qr[2], qr[4])
+    ghz_circ.cx(qr[2], qr[1])
+    ghz_circ.measure(qr[2], cr[0])
+    ghz_circ.measure(qr[4], cr[1])
+    ghz_circ.measure(qr[1], cr[2])
+    return meas_calibs, mit_pattern, ghz_circ
+
+
+def meas_calibration_circ_execution(shots: int, seed: int):
+    """
+    create measurement calibration circuits and simulates them with noise
+    Args:
+        shots (int): number of shots per simulation
+        seed (int): the seed to use in the simulations
+
+    Returns:
+        list: list of Results of the measurement calibration simulations
+        list: list of all the possible states with this amount of qubits
+        dict: dictionary of results counts of bell circuit simulation with measurement errors
+    """
+    # define the circuits
+    meas_calibs, state_labels, ghz = meas_calib_circ_creation()
+
+    # define noise
+    prob = 0.2
+    error_meas = pauli_error([('X', prob), ('I', 1 - prob)])
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(error_meas, "measure")
+
+    # run the circuits multiple times
+    backend = qiskit.Aer.get_backend('qasm_simulator')
+    cal_results = qiskit.execute(meas_calibs, backend=backend, shots=shots, noise_model=noise_model,
+                                 seed_simulator=seed).result()
+
+    bell_results = qiskit.execute(ghz, backend=backend, shots=shots, noise_model=noise_model,
+                                  seed_simulator=seed).result().get_counts()
+
+    return cal_results, state_labels, bell_results
+
+
+def tensored_calib_circ_execution(shots: int, seed: int):
+    """
+    create tensored measurement calibration circuits and simulates them with noise
+    Args:
+        shots (int): number of shots per simulation
+        seed (int): the seed to use in the simulations
+
+    Returns:
+        list: list of Results of the measurement calibration simulations
+        list: the mitigation pattern
+        dict: dictionary of results counts of bell circuit simulation with measurement errors
+    """
+    # define the circuits
+    meas_calibs, mit_pattern, ghz_circ = tensored_calib_circ_creation()
+    # define noise
+    prob = 0.2
+    error_meas = pauli_error([('X', prob), ('I', 1 - prob)])
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(error_meas, "measure")
+
+    # run the circuits multiple times
+    backend = qiskit.Aer.get_backend('qasm_simulator')
+    cal_results = qiskit.execute(meas_calibs, backend=backend, shots=shots, noise_model=noise_model,
+                                 seed_simulator=seed).result()
+
+    bell_results = qiskit.execute(ghz_circ, backend=backend, shots=shots, noise_model=noise_model,
+                                  seed_simulator=seed).result()
+
+    return cal_results, mit_pattern, bell_results
+
+
+def generate_meas_calibration(results_file_path: str, runs: int):
+    """
+    run the measurement calibration circuits, calculates the fitter matrix in few methods
```

Comment:
Either "run ... calculate" or "runs ... calculates". The first option is more preferable.
Also "in a few methods".